portfolio.html

```diff
@@ -234,6 +234,59 @@ <h1 class="about_me1">About me</h1>
       </div>
     </div>
     </section>
-  
+
+  <!-- Contact form starts from here -->
+  <section class="contact-form-1">
+    <h1 class="about_me1">
+      I'm always interested in hearing about new projects, 
+      so if you'd like to chat please get in touch.
+    </h1>
+  </section>
+
+  <section class="contact-form-2">
+
+        <form action="https://formspree.io/f/xgebkzaz" method="post">
+          <ul>
+
+            <li class="name-box">
+              <label for="name"></label>
+              <input name="user-name" type="text" placeholder="Full name" maxlength="30" class="name" required/>
+            </li>
+
+            <li class="email-box"> 
+              <label for="email"></label>
+              <input type="email" name="email" placeholder="Email address" class="email" required/>
+            </li>
+
+            <li class="text-box">
+              <label for="message"></label>
+              <textarea type="text" name="message" rows="5" placeholder="Enter yout text here"  maxlength="500" class="text" required></textarea>
+            </li>
+          </ul>
+          
+            <div class="b_div">
+              <button class="resume_button">
+                Get in touch
+              </button>
```

Comment:
- [ ] You did well 😃 at adding the form spree to the form, but there is a little change at the button submit, kindly try to add the hover and pressed effect to the button because when I try to click the pressed effect not displaying to me , so please try to make that to be like the Figma Design 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/220443700-9152617b-512a-4098-b31e-ddaff97d26a1.png)  |    ![Recording 2023-02-21 at 21 46 43](https://user-images.githubusercontent.com/83360636/220443636-bf2d4a4f-cee6-40dd-966d-ede5c25a6279.gif)  |